contracts/common.yml

```diff
@@ -5961,6 +5961,10 @@ components:
           type: string
           enum:
             - flux
+        params:
+          type: object
```

Comment:
@glinton, @yquansah 

Why the `params` is not defined as a dictionary? Something like:

```yaml
params:
  type: object
  additionalProperties: true
  description: |
    Enumeration of key/value pairs that respresent parameters to be injected into query (can only specify either this field or extern and not both)
```

- https://swagger.io/docs/specification/data-models/dictionaries/